### PR TITLE
release 1.0: modernise jbuilder usage and remove hvcat package

### PR DESCRIPTION
**Pull request by avsm**
the hvcat binary is bundled with the hvsock opam package now as
it is not enough to be standalone (ok by @djs55)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,4 +14,19 @@ uninstall:
 	jbuilder uninstall
 
 clean:
-	rm -rf _build
+	rm -rf _build *.install
```
Comment (samoht) [inline Makefile]: `jbuilder clean` would be better as it also remove the merlin files

File: Makefile [hunk 2]
```diff
@@ -14,4 +14,19 @@ uninstall:
...
+
+REPO=../../mirage/opam-repository
```
Comment (samoht) [inline Makefile]: that doesn't fit with my repo structure :-)

File: Makefile (hunk 1 above)
Comment (avsm) [inline Makefile]: fixed in 6a1559b99da64d56f409098d4fefd84c7ec2b830

File: src/jbuild [hunk 3]
```diff
@@ -4,5 +4,4 @@
 
 (install
  ((section bin)
-  (package hvcat)
```
Comment (samoht) [inline src/jbuild]: are you sure this is not needed anymore?

Comment (avsm) [inline src/jbuild]: it's not, as there is only one package

**Comment by avsm** [comment]
--dev still fails, so not adding it yet. Need to go around and make the trees more --dev safe overall...

**Pull request merged**